backend/common/src/main/java/ai/verta/modeldb/common/config/Config.java

```diff
@@ -25,18 +25,19 @@
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
 
+@SuppressWarnings({"squid:S116", "squid:S100"})
 public abstract class Config {
-  public static String MISSING_REQUIRED = "required field is missing";
-
-  public ServiceConfig authService;
-  public Map<String, CronJobConfig> cron_job = new HashMap<>();
-  public boolean populateConnectionsBasedOnPrivileges = false;
-  public DatabaseConfig database;
-  public boolean enableTrace = false;
-  public GrpcServerConfig grpcServer;
-  public SpringServerConfig springServer;
-  public ServiceUserConfig service_user;
-  public int jdbi_retry_time = 100; // Time in ms
+  public static final String MISSING_REQUIRED = "required field is missing";
+
+  private ServiceConfig authService;
+  private Map<String, CronJobConfig> cron_job = new HashMap<>();
+  private boolean populateConnectionsBasedOnPrivileges = false;
+  private DatabaseConfig database;
+  private boolean enableTrace = false;
+  private GrpcServerConfig grpcServer;
+  private SpringServerConfig springServer;
+  private ServiceUserConfig service_user;
+  private int jdbi_retry_time = 100; // Time in ms
```

Comment:
since you're updating these, make them camel case with no underscores, like `jdbiRetryTime`

Reply:
We can not use camel case here because our config file used the underscore pattern which we can not change so

Reply:
sounds good 👍🏼